core/package.json

```diff
@@ -23,6 +23,285 @@
   "collection:main": "dist/collection/index.js",
   "collection": "dist/collection/collection-manifest.json",
   "types": "dist/types/interface.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs.js",
+      "types": "./dist/types/interface.d.ts"
+    },
+    "./ion-accordion-group": {
```

Comment:
I'd recommend adding `/components/` to the path to minimize the changes to the Vue and React packages (and anyone consuming the custom elements bundles from `@ionic/core`).